tests/datavolume_test.go

```diff
@@ -1092,8 +1095,8 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 		})
 
 		It("Should create a new PVC when PVC is deleted during import", func() {
-			dataVolumeSpec := createRegistryImportDataVolume(dataVolumeName, "1Gi", tinyCoreIsoRegistryURL())
-			By(fmt.Sprintf("creating new datavolume %s", dataVolumeSpec.Name))
+			dataVolumeSpec := utils.NewDataVolumeWithHTTPImport(dataVolumeName, "1Gi", httpTinyCoreRateLimit())
```

Comment:
Wasn't the point of this test to delete during a registry import?

Reply:
This was a test for https://issues.redhat.com/browse/CNV-5149. I needed to replace registry import with rate-limited http import, because the registry import was too fast and race conditions started to occur, making the test unreliable.

Reply:
As I said that test is specifically for registry imports, so if the registry import is too fast now (that is great btw), then we need to figure out some rate limiting for the registry. Maybe we can combine nginx as a proxy for the registry? Some Googling found people doing that.